inventory/sample/group_vars/all/all.yml

```diff
@@ -55,6 +55,11 @@ loadbalancer_apiserver_healthcheck_port: 8081
 ## TODO(riverzhang): https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager
```

Comment:
Can you remove the note about the external cloud provider not being supported?

Reply:
Thanks for the review.
Yes I should, but I think that there should be some work that the kubeadm conf should be updated with cloud provider external. I'll check that and update this PR.